org-ros.el

```diff
@@ -43,24 +43,47 @@
 (defcustom org-ros-secondary-screencapture-switch "-s"
   "Secondary screencapture software selection switch."
   :type 'string)
- 
+
+(defcustom org-ros-windows-screencapture "powershell.exe"
+  "Powershell screencapture software (for people using wsl2)."
```

Comment:
Maybe you forgot this?

(defcustom org-ros-windows-screencapture "powershell.exe"
"Powershell screencapture software (for those using Windows or WSL)."

Reply:
done!